docs/develop/php/index.mdx

```diff
@@ -25,7 +26,7 @@ Build Temporal Applications with the PHP SDK.
 
 - [Temporal PHP Community Slack](https://temporalio.slack.com/archives/C01LK9FAMM0)
 - [PHP SDK Forum](https://community.temporal.io/tag/php-sdk)
-:::
+  :::
```

Comment:
This needs to be left aligned and not aligned with the preceding bullet. It also needs an empty line before it. and line 16 needs an empty line after.